# adcSensorStamped msg
Header header
int16 adc_sensor_1
int16 adc_sensor_2

